apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -0,0 +1,298 @@
+            </div>
+            <div>
+              <FormItem className="w-full">
+                <div className="flex items-center">
+                  <FormLabel
+                    tooltip="Provide a unique ID for the user as the subscriberId (e.g., your app's internal user ID)."
+                    className="gap-1"
+                  >
+                    SubscriberId
+                  </FormLabel>
+                  <span className="ml-auto">
+                    <Link
+                      to="https://docs.novu.co/concepts/subscribers"
+                      className="text-xs font-medium text-neutral-600 hover:underline"
+                      target="_blank"
+                    >
+                      How it works?
+                    </Link>
+                  </span>
+                </div>
+                <Input value={subscriberId} readOnly trailingNode={<CopyButton valueToCopy={subscriberId} />} />
+              </FormItem>
+            </div>
+
+            <div className="flex flex-1 items-center gap-2.5">
+              <FormField
+                control={form.control}
+                name="email"
+                render={({ field, fieldState }) => (
+                  <FormItem className="w-full">
+                    <FormLabel>Email address</FormLabel>
+                    <FormControl>
+                      <Input
+                        {...field}
+                        type="email"
+                        placeholder={field.name}
+                        id={field.name}
+                        value={field.value || undefined}
+                        onChange={field.onChange}
+                        hasError={!!fieldState.error}
+                      />
+                    </FormControl>
+                  </FormItem>
+                )}
+              />
+              <FormField
+                control={form.control}
+                name="phone"
+                render={({ field }) => (
+                  <FormItem className="w-full">
+                    <FormLabel>Phone number</FormLabel>
+                    <FormControl>
+                      <PhoneInput
+                        {...field}
+                        placeholder={field.name}
+                        id={field.name}
+                        value={field.value || ''}
+                        onChange={field.onChange}
+                      />
+                    </FormControl>
+                  </FormItem>
+                )}
+              />
+            </div>
+            <Separator />
+
+            <div className="flex flex-1 items-center gap-2.5">
+              <FormField
+                control={form.control}
+                name="locale"
+                render={({ field }) => (
+                  <FormItem className="w-1/4">
+                    <FormLabel>Locale</FormLabel>
+                    <FormControl>
+                      <TimezoneSelect {...field} value={field.value} onValueChange={field.onChange} />
```

Comment:
Should these 2 fields show the same "Select a timezone" label?

![image](https://github.com/user-attachments/assets/30afc3e5-4fbf-4848-b22e-eeac8c5086b0)

Reply:
Here it is just a dummy component, The locale dropdown is implemented in this PR: https://github.com/novuhq/novu/pull/7635